### DISCUSSION TITLE
Recording Requests with CLI

### DISCUSSION THREAD

**Discussion by gingerwizard** [mockoon/mockoon#589] [Ideas] [answered]
We're looking to use Mockoon for a complex web application, where results are potentially executed in various order. The number of requests are potentially significant and recording these using the UI isn't feasible - especially since we need to match on the body, as many different requests go to the same endpoints (think REST datastore queries). Achieving this with the current UI workflow simply isn't realistic.

As an alternative, we would like to bulk record all requests using the CLI - using the same proxy feature. We could then theoretically convert these for serving as mock requests and exploit the CLI per normal.

Are there any plans to implement this? How have others mocked very large request volumes?

**Comment by 255kb** [comment, marked as answer]
The CLI has exactly the same features than the UI: proxy mode, etc. Only big difference is that the logged requests in the CLI do not contain much information (method, URL, status code). There is an [opened issue](https://github.com/mockoon/cli/issues/34) asking for multiple things, but mainly to have more info in the logs, like the body, headers, etc. 
As per your question, I am not sure to perfectly understand what you are trying to achieve. I guess you would like to "auto-mock" (i.e. automatically create endpoints from entering requests) using the CLI? This is not supported and is not planned either.
